enh(network-huawei-snmp): added new service macro CTOR-1986 (#4767)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-huawei-wlc-snmp.md

## Connector dependencies

The following monitoring connectors will be installed when you install the **Huawei WLC SNMP** connector through the

> **Configuration > Monitoring Connector Manager** menu:

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu:
```

| Service Alias | Service Template                       | Service Description                             |
|:--------------|:---------------------------------------|:------------------------------------------------|

> | Cpu           | Net-Huawei-Wlc-Cpu-SNMP-custom         | Monitor the utilization rate of the CPU         |

Reviewer: ```suggestion
| Cpu           | Net-Huawei-Wlc-Cpu-SNMP-custom         | Monitor the CPU utilization rate         |
```

| Hardware      | Net-Huawei-Wlc-Hardware-SNMP-custom    | Check the state of hardware components          |

> | Memory        | Net-Huawei-Wlc-Memory-SNMP-custom      | Check the rate of the utilization of memory     |

Reviewer: ```suggestion
| Memory        | Net-Huawei-Wlc-Memory-SNMP-custom      | Check the memory utilization rate  |
```

### Pack

1. If the platform uses an *online* license, you can skip the package installation
instruction below as it is not required to have the connector displayed within the

> **Configuration > Monitoring Connector Manager** menu.

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu.
```

</Tabs>

2. Whatever the license type (*online* or *offline*), install the **Huawei WLC SNMP** connector through

> the **Configuration > Monitoring Connector Manager** menu.

Reviewer: ```suggestion
the **Configuration > Connectors > Monitoring Connectors** menu.
```

| WARNINGSTATUS                    | Define the conditions to match for the status to be WARNING. (default: C\<'%\{runstate\} =~ /countryCodeMismatch\|typenotmatch/'\>). You can use the following variables: C\<%\{runstate\}\>, C\<%\{display\}\>. C\<%(runstate)\> can have one of these values: C\<idle\>, C\<autofind\>, C\<typenotmatch\>, C\<fault\>, C\<config\>, C\<configFailed\>, C\<download\>, C\<normal\>, C\<committing\>, C\<commitFailed\>, C\<standby\>, C\<verMismatch\>, C\<nameConflicted\>, C\<invalid\>, C\<countryCodeMismatch\>                                        |               |           |

> | CRITICALSTATUS                   | Define the conditions to match for the status to be CRITICAL. (default: C\<'%\{runstate\} =~ /fault\|configFailed\|commitFailed\|verMismatch\|nameConflicted\|invalid/'\>). You can use the following variables: C\<%\{runstate\}\>, C\<%\{display\}\>. C\<%(runstate)\> can have one of these values: C\<idle\>, C\<autofind\>, C\<typenotmatch\>, C\<fault\>, C\<config\>, C\<configFailed\>, C\<download\>, C\<normal\>, C\<committing\>, C\<commitFailed\>, C\<standby\>, C\<verMismatch\>, C\<nameConflicted\>, C\<invalid\>, C\<countryCodeMismatch\> |               |           |

Reviewer: C

> | CRITICALSTATUS                   | Define the conditions to match for the status to be CRITICAL. (default: C\<'%\{runstate\} =~ /fault\|configFailed\|commitFailed\|verMismatch\|nameConflicted\|invalid/'\>). You can use the following variables: C\<%\{runstate\}\>, C\<%\{display\}\>. C\<%(runstate)\> can have one of these values: C\<idle\>, C\<autofind\>, C\<typenotmatch\>, C\<fault\>, C\<config\>, C\<configFailed\>, C\<download\>, C\<normal\>, C\<committing\>, C\<commitFailed\>, C\<standby\>, C\<verMismatch\>, C\<nameConflicted\>, C\<invalid\>, C\<countryCodeMismatch\> |               |           |

Author: ```suggestion
| WARNINGSTATUS                    | Define the conditions to match for the status to be WARNING. (default: `'%\{runstate\} =~ /countryCodeMismatch\|typeNotMatch/'`). You can use the following variables: `%\{runstate\}`, `%\{display\}`. `%(runstate)` can have one of these values: `idle`, `autofind`, `typeNotMatch`, `fault`, `config`, `configFailed`, `download`, `normal`, `committing`, `commitFailed`, `standby`, `verMismatch`, `nameConflicted`, `invalid`, `countryCodeMismatch`                                        |                   |             |
| CRITICALSTATUS                   | Define the conditions to match for the status to be CRITICAL. (default: `'%\{runstate\} =~ /fault\|configFailed\|commitFailed\|verMismatch\|nameConflicted\|invalid/'`). You can use the following variables: `%\{runstate\}`, `%\{display\}`. `%(runstate)` can have one of these values: `idle`, `autofind`, `typeNotMatch`, `fault`, `config`, `configFailed`, `download`, `normal`, `committing`, `commitFailed`, `standby`, `verMismatch`, `nameConflicted`, `invalid`, `countryCodeMismatch` |                   |             |
```

| OIDFILTER                 | Define the OID to be used to filter interfaces (values: ifDesc, ifAlias, ifName, IpAddr)                                                                            | ifname                                                    |           |
| OIDDISPLAY                | Define the OID that will be used to name the interfaces (values: ifDesc, ifAlias, ifName, IpAddr)                                                                   | ifname                                                    |           |
| INTERFACENAME             | Check only the interfaces with the specified IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored. To filter on interface names, see --name |                                                           |           |
| WARNINGBIASCURRENT        | Thresholds in C\<mA\>                                                                                                                                               |                                                           |           |

> | CRITICALBIASCURRENT       | Thresholds in C\<mA\>                                                                                                                                               |                                                           |           |

Author: ```suggestion
| WARNINGBIASCURRENT        | Thresholds in `mA`                                                                                                                                               |                                                           |           |
| CRITICALBIASCURRENT       | Thresholds in `mA`                                                                                                                                               |                                                           |           |
```

| CRITICALINDISCARD         | Thresholds                                                                                                                                                          |                                                           |           |
| WARNINGINERROR            | Thresholds                                                                                                                                                          |                                                           |           |
| CRITICALINERROR           | Thresholds                                                                                                                                                          |                                                           |           |
| WARNINGINPUTPOWER         | Thresholds in C\<dBm\>                                                                                                                                              |                                                           |           |

> | CRITICALINPUTPOWER        | Thresholds in C\<dBm\>                                                                                                                                              |                                                           |           |

Author: ```suggestion
| WARNINGINPUTPOWER         | Thresholds in `dBm`                                                                                                                                              |                                                           |           |
| CRITICALINPUTPOWER        | Thresholds in `dBm`                                                                                                                                              |                                                           |           |
```

| CRITICALOUTDISCARD        | Thresholds                                                                                                                                                          |                                                           |           |
| WARNINGOUTERROR           | Thresholds                                                                                                                                                          |                                                           |           |
| CRITICALOUTERROR          | Thresholds                                                                                                                                                          |                                                           |           |
| WARNINGOUTPUTPOWER        | Thresholds in C\<dBm\>                                                                                                                                              |                                                           |           |

> | CRITICALOUTPUTPOWER       | Thresholds in C\<dBm\>                                                                                                                                              |                                                           |           |

Reviewer: C

> | CRITICALOUTPUTPOWER       | Thresholds in C\<dBm\>                                                                                                                                              |                                                           |           |

Author: ```suggestion
| WARNINGOUTPUTPOWER        | Thresholds in `dBm`                                                                                                                                              |                                                           |           |
| CRITICALOUTPUTPOWER       | Thresholds in `dBm`                                                                                                                                              |                                                           |           |
```

| --filter-group                         |   Filter access point group (can be a regexp).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
| --warning-status                       |   Define the conditions to match for the status to be WARNING. (default: C\<'%\{runstate\} =~ /countryCodeMismatch\|typenotmatch/'\>). You can use the following variables: C\<%\{runstate\}\>, C\<%\{display\}\>. C\<%(runstate)\> can have one of these values: C\<idle\>, C\<autofind\>, C\<typenotmatch\>, C\<fault\>, C\<config\>, C\<configFailed\>, C\<download\>, C\<normal\>, C\<committing\>, C\<commitFailed\>, C\<standby\>, C\<verMismatch\>, C\<nameConflicted\>, C\<invalid\>, C\<countryCodeMismatch\>.                                          |

> | --critical-status                      |   Define the conditions to match for the status to be CRITICAL. (default: C\<'%\{runstate\} =~ /fault\|configFailed\|commitFailed\|verMismatch\|nameConflicted\|invalid/'\>). You can use the following variables: C\<%\{runstate\}\>, C\<%\{display\}\>. C\<%(runstate)\> can have one of these values: C\<idle\>, C\<autofind\>, C\<typenotmatch\>, C\<fault\>, C\<config\>, C\<configFailed\>, C\<download\>, C\<normal\>, C\<committing\>, C\<commitFailed\>, C\<standby\>, C\<verMismatch\>, C\<nameConflicted\>, C\<invalid\>, C\<countryCodeMismatch\>.   |

Reviewer: C

> | --critical-status                      |   Define the conditions to match for the status to be CRITICAL. (default: C\<'%\{runstate\} =~ /fault\|configFailed\|commitFailed\|verMismatch\|nameConflicted\|invalid/'\>). You can use the following variables: C\<%\{runstate\}\>, C\<%\{display\}\>. C\<%(runstate)\> can have one of these values: C\<idle\>, C\<autofind\>, C\<typenotmatch\>, C\<fault\>, C\<config\>, C\<configFailed\>, C\<download\>, C\<normal\>, C\<committing\>, C\<commitFailed\>, C\<standby\>, C\<verMismatch\>, C\<nameConflicted\>, C\<invalid\>, C\<countryCodeMismatch\>.   |

Author: ```suggestion
| --warning-status                       | Define the conditions to match for the status to be WARNING. (default: `'%\{runstate\} =~ /countryCodeMismatch\|typeNotMatch/'`). You can use the following variables: `%\{runstate\}`, `%\{display\}`. `%(runstate)` can have one of these values: `idle`, `autofind`, `typeNotMatch`, `fault`, `config`, `configFailed`, `download`, `normal`, `committing`, `commitFailed`, `standby`, `verMismatch`, `nameConflicted`, `invalid`, `countryCodeMismatch`.                                        |
| --critical-status                      | Define the conditions to match for the status to be CRITICAL. (default: `'%\{runstate\} =~ /fault\|configFailed\|commitFailed\|verMismatch\|nameConflicted\|invalid/'`). You can use the following variables: `%\{runstate\}`, `%\{display\}`. `%(runstate)` can have one of these values: `idle`, `autofind`, `typeNotMatch`, `fault`, `config`, `configFailed`, `download`, `normal`, `committing`, `commitFailed`, `standby`, `verMismatch`, `nameConflicted`, `invalid`, `countryCodeMismatch`. |
```

| --warning-bias-current                          |   Thresholds in C\<mA\>.                                                                                                                                                                                                                                                                     |
| --critical-bias-current                         |   Thresholds in C\<mA\>.                                                                                                                                                                                                                                                                     |
| --warning-output-power                          |   Thresholds in C\<dBm\>.                                                                                                                                                                                                                                                                    |

> | --critical-output-power                         |   Thresholds in C\<dBm\>.                                                                                                                                                                                                                                                                    |

Reviewer: C

> | --critical-output-power                         |   Thresholds in C\<dBm\>.                                                                                                                                                                                                                                                                    |

Author: ```suggestion
| --warning-input-power                           |   Thresholds in `dBm`.                                                                                                                                                                                                                                                                    |
| --critical-input-power                          |   Thresholds in `dBm`.                                                                                                                                                                                                                                                                    |
| --warning-bias-current                          |   Thresholds in `mA`.                                                                                                                                                                                                                                                                     |
| --critical-bias-current                         |   Thresholds in `mA`.                                                                                                                                                                                                                                                                     |
| --warning-output-power                          |   Thresholds in `dBm`.                                                                                                                                                                                                                                                                    |
| --critical-output-power                         |   Thresholds in `dBm`.                                                                                                                                                                                                                                                                    |
```
